packages/strapi-plugin-content-type-builder/services/schema-builder/content-type-builder.js

```diff
@@ -158,7 +158,8 @@ module.exports = function createComponentBuilder() {
             this.unsetRelation(oldAttribute);
           }
 
-          newAttribute.autoPopulate = newAttribute.autoPopulate || oldAttribute.autoPopulate;
+          // keep extra options that were set manually on oldAttribute
+          _.defaults(newAttribute, oldAttribute);
```

Comment:
Is it a bug fix too? We couldn't disable the autoPopulate before?

Reply:
just not required anymore as it is implement with the _.defaults under it :)

Reply:
But this:
```js
newAttribute.autoPopulate = newAttribute.autoPopulate || oldAttribute.autoPopulate;
```
Would always be true if it was true in the old attribute right ?

Reply:
Yes but it cannot be defined in the newAttribute the code was made just to keep the value if it was set in the oldAttribute so the _defaults is good enough to replace it :)